### ISSUE TITLE
Determine url/hosting for Thimble

### ISSUE THREAD

**Issue by hannahkane** [MozillaFoundation/plan#519]
We need to move thimble off of Webmaker, so this likely means moving to thimble.mozilla.org. I don't have an understanding of the technical and security implications involved. 

@simonwex to advise about what the work is here

**Comment by simonwex** [comment]
@hannahkane I'll chat to David about this today during our 1:1. Let's catch up tomorrow.

**Comment by hannahkane** [comment]
ok. sent you a calendar invite.

**Comment by simonwex** [comment]
Hey @hannahkane we can use thimble.mozilla.org here. 

@jbuck could you point me at the relevant issues for getting thimble into production and file the request for the DNS?

**Comment by jbuck** [comment]
@simonwex https://github.com/MozillaFoundation/mofo-devops/issues/93

**Comment by hannahkane** [comment]
Question: will we move "old" thimble to a url like thimble-old.mozilla.org and maintain it there for awhile? For the beta, we want users to be able to toggle between old and new.

**Comment by simonwex** [comment]
@humphd what's the best way to support both thimbles in your mind?

**Comment by hannahkane** [comment]
@humphd - ping! want to make sure this doesn't get lost.

**Comment by simonwex** [comment]
@sedge is going to update today channeling humph.

**Comment by sedge** [comment]
**tl;dr**: We want to deprecate old thimble and migrate everything to the new one. With this option the work is minimal and the path forward is clear. 

I think we have two main options to consider. 

##### *We migrate projects en-mass from thimble-classic to thimble-beta*

The main downside to this approach is that we would probably have to do some UI work to simplify brackets for single-file users. Also, people don't always like change. 

It happens to be more straightforward since we're at least somewhat familiar with the UI-specific codebase. It's likely to be the most reasonable option in terms of hours devving.

##### *We fake as many old Webmaker services as possible, and patch thimble-classic to use id.wm.o*

The trouble with the second option is that it's a huge amount of work, because I'm almost certain we don't want to keep the old `Webmaker` branding. And we *definitely* don't want to be running two sets of tools. I'm not sure that an expert on thimble-classic code exists anymore, so I don't feel confident calling out how long the technical side will take. 

This probably means too long... :neutral_face:   

Another challenge that I touched on earlier is that we don't want two APIs for accessing projects for the same gallery. This means we'd deprecate `makeAPI` and have to migrate the makes to live on `publish.wm.o` *anyway*. We'd also have to teach `publish.wm.o` how to serve them in a way thimble-classic will understand. 

It just smells like too much to me, but I'll let you guys make the final call.

cc: @simonwex @humphd @hannahkane @flukeout @Pomax @gideonthomas

**Comment by jbuck** [comment]
@hannahkane I need lead-time to obtain an SSL certificate and DNS changes for thimble.mozilla.org. Can you confirm that this is the URL you'll be using for launch?

**Comment by hannahkane** [comment]
@jbuck - confirmed. Is there not enough time?

**Comment by jbuck** [comment]
@hannahkane Nah, should be okay time-wise

**Comment by davidascher** [comment]
I vote for en-masse migration.

Do we have data to suggest that people will have a problem w/ new thimble?

How many people use old thimble per day today?

**Comment by hannahkane** [comment]
I'm +1 on this, too, as long as old projects will still work in the new Thimble.

**Comment by humphd** [comment]
We have a plan, which I'll summarize here, and file specific tickets in Thimble:
* https://thimble.webmaker.org will remain what it is today, and we'll add some UI to get you to the new Thimble if you so choose.  Old projects will live on there, until we migrate them later, and deprecate this.
* https://thimble.mozilla.org will be where you go for the new Homepage (the `/`  route) and Editor (the `/editor` route). 
* @jbuck has initiated the creation of the new thimble.mozilla.org domain in https://github.com/MozillaFoundation/mofo-devops/issues/157

**Comment by hannahkane** [comment]
I think we can close this ticket. Decisions have been made; remaining work is ticketed elsewhere.
